modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -106,7 +108,7 @@ spec:
 +
 [source,terminal]
 ----
-$ oc apply -f <filename>.yaml
+$ oc apply -f /tmp/40-worker-custom-journald.yaml
```

Comment:
Because you are providing an example value here, you should update line 107 to read:
`. Create the machine config. For example: